src/Admin/FieldDescription.php

```diff
@@ -20,11 +20,6 @@
  */
 class FieldDescription extends BaseFieldDescription
 {
-    public function __construct()
-    {
-        $this->parentAssociationMappings = [];
```

Comment:
This is already the default value. And the __construct() override disallow to pass a name as argument.